packages/components/src/Input/Input.js

```diff
@@ -0,0 +1,57 @@
+import { forwardRef } from 'react';
+import { createStyleSheet } from '@platzily-ui/styling';
+import PropTypes from 'prop-types';
+
+const useStyleSheet = createStyleSheet(
+  (theme, { width, height }) => ({
+    inputDefault: {
+      width: width || 199,
```

Comment:
Instead of doing this why dont we pass a new class when the button is of type checkbox? :thinking: